src/ledger/LedgerTxn.cpp

```diff
@@ -83,7 +83,8 @@ EntryIterator::getImpl() const
     return mImpl;
 }
 
-EntryIterator& EntryIterator::operator++()
```

Comment:
looks like you are using different version of clang-format that everyone

Reply:
Fixing, it looks like I was using 7 instead of 5 -- thanks!